src/platform/linux/x11/mod.rs

```diff
@@ -406,7 +406,7 @@ impl EventsLoop {
                             for i in 0..xev.valuators.mask_len*8 {
                                 if ffi::XIMaskIsSet(mask, i) {
                                     if let Some(&mut (_, ref mut info)) = physical_device.scroll_axes.iter_mut().find(|&&mut (axis, _)| axis == i) {
-                                        let delta = (unsafe { *value } - info.position) / info.increment;
+                                        let delta = (unsafe { *value } - info.position) / info.increment; // TODO: Doesn't handle underflow/overflow
```

Comment:
Is overflow a realistic concern here? The range of double is astronomical, and we're only computing differences between successive values delivered by X.